workspaces/liferay-testray-workspace/client-extensions/liferay-testray-custom-element/src/components/Form/Renderer/Renderer.tsx

```diff
@@ -183,7 +173,7 @@ const Renderer: React.FC<RendererProps> = ({
 
 				const getOptions = () => {
 					const _options =
-						fieldOptions[name] ||
+						filderOptions[name] ||
```

Comment:
```suggestion
						fieldOptions[name] ||
```

Reply:
Resolved in  [[ISSD-2926 Rename](https://github.com/is-solutions-delivery/liferay-portal/pull/5504/commits/bcfa0cdf314ff4dec650821c669f85bc517c8974)]